docs/pages/reference/session-recording.mdx

```diff
@@ -0,0 +1,162 @@
+---
+title: Teleport Session Recording
+description: An overview of Teleport's session recording and its configuration
+---
+
+This page provides an overview of how Teleport records sessions and the various
```

Comment:
We might want to expand on what session recording is earlier in the page. What is a session? What does it mean to record it?

Reply:
Perhaps I'm blinded by my own experience, as it's pretty obvious to me. Do you have any suggestions for what we might say in an expanded explanation?